spec/browser_spec.rb

```diff
@@ -79,16 +80,16 @@
     after { subject.quit }
   end
 
-  describe ".open" do
-    context "when given a block" do
+  describe '.open' do
+    context 'when given a block' do
       it "must yield a new #{described_class} instance" do
-        expect { |b|
+        expect do |b|
           subject.open(&b)
-        }.to yield_successive_args(described_class::Agent)
+        end.to yield_successive_args(described_class::Agent)
```

Comment:
I **strongly** disagree with this rubocop rule. When chaining together multiple methods that take a block, curly brace blocks are the defacto standard. I really have no clue how that rule got into rubocop in the first place, and wish they would remove it.